### DISCUSSION TITLE
Cannot fine env_file and sample valheim.env file

### DISCUSSION THREAD

**Discussion by punksinatra** [lloesche/valheim-server-docker#269] [Q&A]
Hi I am using the [docker compose yml file](https://github.com/lloesche/valheim-server-docker/blob/main/docker-compose.yaml)  via portainer but during initial start there is an error could not find env_file

Do I need to create the file first? Anyone have a sample env file or is it just literally the environment variables inside?

**Comment by lloesche** [comment]
https://github.com/lloesche/valheim-server-docker#deploying-with-docker-compose

**Comment by tpups** [comment]
I had the same issue on an ubuntu 20.04 VM.

Resolved by changing the path for valheim.env from 

`$HOME/valheim-server/valehim.env` to `valheim.env`

**Reply by ellingit** [reply]
I know this is an old thread, but this totally makes sense and fixed the issue for me. The location of the env file should be relative to the docker-compose.yaml file...that's why changing it worked.
